docs/docs-beta/sidebars.ts

```diff
@@ -86,135 +86,47 @@ const sidebars: SidebarsConfig = {
     },
   ],
   integrations: [
+    'integrations/index',
     {
       type: 'category',
-      label: 'Categories',
+      label: 'Guides',
+      collapsed: false,
+      items: [
+        'integrations/guides/multi-asset-integration'
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Libraries',
       collapsible: false,
+      link: {type: 'doc', id: 'integrations/libraries/index'},
       items: [
         {
           type: 'category',
-          label: 'ETL',
-          items: [
-            'integrations/airbyte',
-            'integrations/sdf',
-            'integrations/fivetran',
-            'integrations/dlt',
-            'integrations/census',
-            'integrations/dbt',
-            'integrations/dbt-cloud',
-            'integrations/sling',
-            'integrations/hightouch',
-            'integrations/meltano',
-          ],
-        },
-        {
-          type: 'category',
-          label: 'Storage',
-          items: [
-            'integrations/snowflake',
-            'integrations/gcp/bigquery',
-            'integrations/aws/athena',
-            'integrations/aws/s3',
-            'integrations/duckdb',
-            'integrations/deltalake',
-            'integrations/aws/redshift',
-            'integrations/gcp/gcs',
-            'integrations/azure-adls2',
-            'integrations/lakefs',
-          ],
-        },
-        {
-          type: 'category',
-          label: 'Compute',
-          items: [
-            'integrations/kubernetes',
-            'integrations/spark',
-            'integrations/aws/glue',
-            'integrations/jupyter',
-            'integrations/aws/emr',
-            'integrations/databricks',
-            'integrations/aws/lambda',
-            'integrations/docker',
-            'integrations/shell',
-            'integrations/gcp/dataproc',
-          ],
-        },
-        {
-          type: 'category',
-          label: 'BI',
-          items: ['integrations/looker'],
-        },
-        {
-          type: 'category',
-          label: 'Monitoring',
-          items: ['integrations/prometheus', 'integrations/datadog', 'integrations/aws/cloudwatch'],
-        },
-        {
-          type: 'category',
-          label: 'Alerting',
+          label: 'Dagster-supported',
```

Comment:
What do you think about making `Dagster-supported` the default, so it's at the top level, and then community supported being a sub folder? I think that might reduce the visual clutter slightly.

Reply:
And then we can create some cool React components to have tags on the integration pages themselves with `dagster verified` or something along those lines.

Reply:
I think that's a great idea! Looks like Docusaurus has tagging built in, so I just used that. All integrations libraries now live in the "Libraries" section and have relevant tags (dagster-supported, community-supported, ETL, etc); tags are at the bottom of the page.